api/schema/project.go

```diff
@@ -127,11 +126,17 @@ type UpdateProjectPatchInput struct {
 	DevelopmentBuildPriority     *uint                `json:"developmentBuildPriority,omitempty"`
 	DeploymentsDisabled          *uint                `json:"deploymentsDisabled,omitempty"`
 	// `null` is valid graphql, use a pointer to allow `nil` to be empty
-	BuildImage *null.String `json:"buildImage,omitempty"`
+	BuildImage              *string `json:"buildImage,omitempty"`
```

Comment:
There is a reason this was `*null.String` as it allows for a JSON value `null` to be provided in the mutation

`{ buildImage: null }` is different to `{ buildImage: "null" }` and the `null.String` allows you to use the first one to effectively "unset" a build image.

Reply:
All updated now